modules/misc/version.nix

```diff
@@ -5,7 +5,7 @@ with lib;
 {
   options = {
     home.stateVersion = mkOption {
-      type = types.enum [ "18.09" "19.03" ];
+      type = types.enum [ "18.09" "19.03" "19.09" ];
       default = "18.09";
```

Comment:
are we not changing the default stateVersion yet?

Reply:
No, the default version will never change. Or at least it will not change for a long time. The idea being that a functioning configuration should not break and changing the state version is always a conscious decision of the user.

Reply:
One could argue that the state version is completely separate from the release version so it becomes a bit confusing that they have the same appearance 😕

Reply:
I see. Thank you for the clarification @rycee!